src/stringIndices.js

```diff
@@ -0,0 +1,13 @@
+const readlineSync = require('readline-sync')
+
+const collectionOfWords = readlineSync.question('Write a funny sentence\n')
+const newCollection = collectionOfWords.split(' ')
```

Comment:
For more clarity, perhaps renaming:

`collectionOfWords` to `userInputString`
and
`newCollection` to `userInputArray`